drivers/char/tpm/tpm_tis_spi_main.c

```diff
@@ -232,6 +234,37 @@ static int tpm_tis_spi_probe(struct spi_device *dev)
 	return tpm_tis_spi_init(dev, phy, irq, &tpm_spi_phy_ops);
 }
 
+static int tpm_tis_spi_parse_dt(struct device *dev)
+{
+	int gpio;
+	int rc = 0;
+
+	gpio = of_get_named_gpio_flags(dev->of_node, "reset-gpio", 0, NULL);
+	if (gpio == -EPROBE_DEFER)
+		return gpio;
+
+	if (gpio_is_valid(gpio)) {
+		rc = devm_gpio_request_one(dev, gpio, GPIOF_OUT_INIT_LOW,
+					   "slb9670_rst_n");
+		if (rc) {
+			dev_err(dev, "failed to get reset-gpio %d\n", gpio);
```

Comment:
As of the comment above you might write here `reset gpio` so it doesn't imply the name4 of the of property.